### PR TITLE
Run workflow cps with Caffeine cache and unignore related tests

### PR DESCRIPTION
**Pull request by bitwiseman**
Test-only for https://github.com/jenkinsci/script-security-plugin/pull/160

### CODE REVIEW

**Comment by jglick** [comment]
[JENKINS-45047](https://issues.jenkins-ci.org/browse/JENKINS-45047), sigh.

By the way the GH feature to create a draft PR is nice for this sort of thing.

**Comment by kshultzCB** [comment]
Truth. A short descripton [here](https://github.com/bitwiseman/workflow-cps-plugin/pull/1).

**Comment by bitwiseman** [comment]
Thank you very much @kshultzCB!

File: pom.xml [hunk 1]
```diff
@@ -97,7 +97,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>script-security</artifactId>
-            <version>1.58</version>
+            <version>1.60-rc860.42e743108fee</version>
```
Comment (dwnusbaum) [inline pom.xml]: 1.61 was just released.
```suggestion
            <version>1.61</version>
```

**Comment by dwnusbaum** [comment]
Windows agent was dead, rebuilding.

**Pull request merged**